### PR TITLE
Fix #1933 Converted StorageUtils to Kotlin

### LINKED ISSUE #1933: Converting StorageUtils to Kotlin
**Is your feature request related to a problem? Please describe.**

StorageUtils is in java. It can be converted to kotlin.

**Describe the solution you'd like**

Convert it to kotlin.

**Additional context**

Parent Issue #1826

### PR DESCRIPTION
**Pull request by akkywadhwa**
Fixes #1933

**Screenshots** 
N/A

### CODE REVIEW

**Comment by macgills** [comment]
```
> Task :core:detekt
Ruleset: style - 5min debt
	UtilityClassWithPublicConstructor - [StorageUtils] at /home/runner/work/kiwix-android/kiwix-android/core/src/main/java/org/kiwix/kiwixmobile/core/utils/StorageUtils.kt:20:1

Overall debt: 5min
```

**Comment by akkywadhwa** [comment]
@macgills I was about to make StorageUtils an object. Should I make another PR for it?

**Comment by macgills** [comment]
I have no idea how I closed this? Reopening

File: core/src/main/java/org/kiwix/kiwixmobile/core/utils/StorageUtils.kt [hunk 1]
```diff
@@ -17,10 +17,7 @@
...
 package org.kiwix.kiwixmobile.core.utils
 
-class StorageUtils {
-  companion object {
-    @JvmStatic
-    fun getFileNameFromUrl(url: String?): String =
-      NetworkUtils.getFileNameFromUrl(url).replace(".meta4", "")
-  }
+object StorageUtils {
+  fun getFileNameFromUrl(url: String?): String =
```
Comment (macgills) [inline core/src/main/java/org/kiwix/kiwixmobile/core/utils/StorageUtils.kt]: mark it JvmStatic so we don't have to use `INSTANCE` to reference it in Kotlin

**Comment by akkywadhwa** [comment]
@macgills I hope everything is good to go in this PR.

**Comment by macgills** [comment]
Yup just forgot to merge it, thanks for the reminder

**Pull request merged**